File: corpus/typescript/fixture_13.ts
type CartItem13 = { price: number; qty?: number };
export function calculateDiscount13(cartItems: CartItem13[], taxRate: number): number {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 130 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook13 {
  lookupPrice(sku: string): number { return sku.length + 13; }
}

File: corpus/python/fixture_13.py
def calculate_discount_13(cart_items, tax_rate):
    total_amount = sum(item["price"] * item.get("qty", 1) for item in cart_items)
    if total_amount > 130:
        return total_amount * (1 + tax_rate)
    return total_amount

class PriceBook13:
    def lookup_price(self, sku):
        return len(sku) + 13
